compiler/optimizer/ExpressionsSimplification.cpp

```diff
@@ -501,10 +513,10 @@ TR_ExpressionsSimplification::invalidateCandidates()
       {
       traceMsg(comp(), "Checking which candidates may be invalidated\n");
 
-      ListIterator<TR::TreeTop> treeTops(_candidateTTs);
-      for (TR::TreeTop *treeTop = treeTops.getFirst(); treeTop; treeTop = treeTops.getNext())
+      ListIterator<SimplificationCandidateTuple> candidates(_candidates);
+      for (SimplificationCandidateTuple *nextCandidate = candidates.getFirst(); nextCandidate; nextCandidate = candidates.getNext())
          {
-         traceMsg(comp(), "   Candidate treetop: %p node: %p\n", treeTop, treeTop->getNode());
```

Comment:
We lost this `traceMsg()` and now the loop doesn't do anything

Reply:
Thanks for catching that!  I had intended to replace it with

```
nextCandidate->print(comp());
```